pkg/operator/resourcesynccontroller/resourcesync_controller.go

```diff
@@ -137,7 +137,6 @@ func (c *ResourceSyncController) sync() error {
 	switch operatorSpec.ManagementState {
 	case operatorv1.Managed:
 	case operatorv1.Unmanaged:
-		return nil
```

Comment:
doesn't this affect more than control plane operators?

Reply:
Should we emit a warning or something? At the least we should add debug logs around what state the operator observes here.

Reply:
added event with warning